pytestqt/exceptions.py

```diff
@@ -70,8 +70,16 @@ def format_captured_exceptions(exceptions):
     message = 'Qt exceptions in virtual methods:\n'
     message += '_' * 80 + '\n'
     for (exc_type, value, tback) in exceptions:
-        message += ''.join(traceback.format_tb(tback)) + '\n'
-        message += '%s: %s\n' % (exc_type.__name__, value)
+        if sys.version_info[0] > 2:
+            from io import StringIO
+            import traceback
+            s = StringIO()
+            traceback.print_exception(type_, value, tback, file=s)
```

Comment:
format_tb hides information on the cause of another exception (so, if an exception is raised from another exception, the cause is lost).